packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -94,7 +94,7 @@ const ALERT_RULE_TAGS = `${ALERT_RULE_NAMESPACE}.tags` as const;
 // kibana.alert.rule_type_id - rule type id for rule that generated this alert
 const ALERT_RULE_TYPE_ID = `${ALERT_RULE_NAMESPACE}.rule_type_id` as const;
 
-// kibana.alert.url - allow our user to go back to the details url in kibana
+// kibana.alert.url - url which will redirect users to the alert page filtered for the given alert
```

Comment:
`the alert page filtered for the given alert`

Since the AO team is working on introducing a dedicated page for each alert, this comment will not be accurate in the future. How about changing it to something like:

`url which will redirect users to a page related to the given alert`

Reply:
Yep, will do, thanks @maryam-saeidi . Are there any tickets for the planned work @maryam-saeidi ?

Reply:
Yes, we have different epics for different rule types, for example:
- https://github.com/elastic/kibana/issues/145117
- https://github.com/elastic/kibana/issues/150854